system/audit-logs-otel/templates/audit-logs-collector.yaml

```
@@ -0,0 +1,401 @@
+apiVersion: opentelemetry.io/v1beta1
+kind: OpenTelemetryCollector
+metadata:
+  name: audit-logs
+  labels:
+  {{- include "plugin.labels" . | nindent 4 }} 
+  {{- if .Values.openTelemetryPlugin.auditLogs.customLabels }}
+  {{ toYaml .Values.openTelemetryPlugin.auditLogs.customLabels | nindent 4 }}
+  {{- end }}
+spec:
+  mode: daemonset
+  tolerations:
+  - operator: Exists
+  podAnnotations:
+    kubectl.kubernetes.io/default-container: "otc-container"
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: cloud.sap/deployment-state
+            operator: NotIn
+            values:
+            - reinstalling
+          - key: cloud.sap/esx-in-maintenance
+            operator: NotIn
+            values:
+            - alarm
+  env:
+    - name: NAMESPACE
+      valueFrom:
+        fieldRef:
+          fieldPath: metadata.namespace
+    - name: KUBE_NODE_NAME
+      valueFrom:
+        fieldRef:
+          fieldPath: spec.nodeName
+    - name: MY_POD_IP
+      valueFrom:
+        fieldRef:
+          apiVersion: v1
+          fieldPath: status.podIP
+    - name: cluster
+      value: "{{ .Values.openTelemetryPlugin.auditLogs.cluster }}"
+    - name: region
+      value: "{{ .Values.openTelemetryPlugin.auditLogs.region }}"
+    - name: index
+      value: "{{ .Values.openTelemetryPlugin.auditLogs.openSearchLogs.index }}"
+  envFrom:
+    - secretRef:
+          name: audit-logs-basic-auth
+{{- if .Values.openTelemetryPlugin.auditLogs.prometheus.podMonitor.enabled }}
+  ports:
+    - name: prometheus
+      port: 9999
+{{- end }}
+  image: {{ index .Values "openTelemetryPlugin" "opentelemetry-operator" "manager" "collectorImage" "repository" }}:{{ index .Values "openTelemetryPlugin" "opentelemetry-operator" "manager" "collectorImage" "tag" }}
```

```suggestion
  image: {{ index .Values "openTelemetryPlugin.auditLogs" "opentelemetry-operator" "manager" "collectorImage" "repository" }}:{{ index .Values "openTelemetryPlugin.auditLogs" "opentelemetry-operator" "manager" "collectorImage" "tag" }}
```
Suggestion: With this we would have the possibility to have separate images for `logs` and `audit-logs`